plugins/SSH.koplugin/main.lua

```diff
@@ -20,6 +20,8 @@ end
 
 local SSH = WidgetContainer:new{
     name = 'SSH',
+    fullname = _("SSH"),
+    description = _([[Allows connecting and transfering files to the device over SSH.]]),
```

Comment:
That'd be transferring, but I'll suggest the plainer

Connect and transfer files to the device using SSH.